addons/portal_rating/static/src/xml/portal_tools.xml

```diff
@@ -37,9 +37,12 @@
                 <hr t-if="two_columns"/>
                 <p t-if="!two_columns"><strong>Details</strong></p>
                 <div class="o_website_rating_progress_bars">
-                    <table class="o_website_rating_progress_table">
+                    <t t-set="hasRatingSelection" t-value="widget.get('rating_value')"/>
+                    <table t-attf-class="o_website_rating_progress_table #{hasRatingSelection and 'o_website_rating_progress_table_has_selection' or ''}">
                         <t t-foreach="widget.get('rating_card_values')['percent']" t-as="percent">
-                            <tr class="o_website_rating_select" t-att-data-star="percent['num']" style="opacity: 1">
+                            <t t-set="isRatingSelected" t-value="percent['num']==widget.get('rating_value')"/>
+                            <tr t-attf-class="o_website_rating_select #{isRatingSelected and 'o_website_rating_table_row_selected' or ''}"
```

Comment:
Indeed, that's better. 
I have changed (not pushed yet) the name to selected_rating (instead of selectedRating). But maybe as it is a javascript template, it is better to use CamelCase ?
(I think I had set that variable mainly to say that it can be interpreted as a boolean i.e. there is no valid rating_value "0" which would be ambiguous but if used in a condition, it suppose the same assumption)

Reply:
You're right it's most often done using snake_case in these templates, including in this files.